api/models/trial.js

```diff
@@ -210,6 +221,17 @@ const Trial = BaseModel.extend({
 
       return discrepancies;
     },
+    is_registered() {
+      // Be aware that this depends on `source_id` to be accurate
+      // So if a trial has a `nct` source but a `pubmed` source_id,
+      // then the problem is with the processor
+      let status;
+      const source = this.sources[this.attributes.source_id] || { type: null };
+      if (source.type !== null) {
+        status = source.type === 'register';
+      }
+      return status;
```

Comment:
Also that comparison does not return `undefined` when there's no source, which is what we want it to do, according to a discussion on earlier commits here. So, when a boolean is not enough, we use local vars :smile: